### PR TITLE
add a gitcoin.co/explorer stale bounty filter

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This PR adds a stale bounty filter titled "Not Picked Up 3+ Days". It enables logged in funders view bounties created by them more than 3 days but which are still in "open" state.

Before:

Screenshot 2020-08-13 at 03 35 44

After:

bounties

Here's a [screen recording](https://drive.google.com/file/d/126OReg3zWGoJYgCXqh2OYrXrt4NWMO0j/view?usp=sharing)

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/7195

### CODE REVIEW

File: app/assets/v2/js/pages/dashboard.js [hunk 1]
```diff
@@ -529,6 +529,18 @@ var refreshBounties = function(event, offset, append) {
   }
 
   explorer.bounties_request = $.get(bountiesURI, function(results, x) {
+
+    // Filter results by open bounties created more than 3 days ago
+    if ($('input[name="bounty_filter"]:checked').val().toString() === 'stale') {
+      results = results.filter(bounty => {
+        let now = new Date();
+        let created = new Date(bounty.web3_created);
+        let daysAgo = new Date(now.setDate(now.getDate() - 3));
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/dashboard.js:538]: ^ update all to `const` ?

Comment (chibie) [reply app/assets/v2/js/pages/dashboard.js:538]: okay sure

**Comment by chibie** [comment]
Thanks for the compliment @thelostone-mc 

The indents were actually altered when I ran `make fix` as specified in the contributing guide.

I'll revert the indents manually as directed.

A video recording would also be attached asap.

**Comment by chibie** [comment]
Hi @thelostone-mc 

All fixes have been made.

File: chatconfig/config.json [hunk 2]
```diff
@@ -0,0 +1,484 @@
+{
```
Comment (thelostone-mc) [inline chatconfig/config.json:1]: @chibie this file still needs to be removed

**Comment by thelostone-mc** [comment]
LGTM apart from that minor removing that file!

File: chatconfig/config.json (hunk 2 above)
Comment (chibie) [reply chatconfig/config.json]: @thelostone-mc okay, I just removed the file.

**Comment by danlipert** [comment]
@chibie nice work! I didn't realize make fix would do that - we should run it against everything sometime

**Pull request merged**